lua/nvim-treesitter/parsers.lua

```diff
@@ -2101,6 +2102,48 @@ list.supercollider = {
   maintainers = { "@madskjeldgaard" },
 }
 
+list.superhtml = {
+  install_info = {
+    url = "https://github.com/kristoff-it/superhtml",
+    includes = { "tree-sitter-superhtml/src" },
+    files = {
+      "tree-sitter-superhtml/src/parser.c",
+      "tree-sitter-superhtml/src/scanner.c",
+    },
+    branch = "main",
+  },
+  filetype = "superhtml",
+  maintainers = { "@rockorager" },
+}
+
+list.supermd = {
+  install_info = {
+    url = "https://github.com/kristoff-it/supermd",
+    includes = { "tree-sitter/supermd/src" },
+    files = {
+      "tree-sitter/supermd/src/parser.c",
```

Comment:
```suggestion
      "src/parser.c",
```
Use the `location` key instead to specify subdirectories.